### PR TITLE
Update deployment-prep-active-directory.md

### PR DESCRIPTION
**Pull request by steveswalwell**
Added the details for the none supported special characters in the OUPath

### CODE REVIEW

**Comment by Court72** [comment]
@alkohli

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/deploy/deployment-prep-active-directory.md [hunk 1]
```diff
@@ -56,6 +56,13 @@ The *AsHciADArtifactsPreCreationTool.ps1* module is used to prepare Active Direc
 |`-AsHciDeploymentPrefix`|The prefix used for all AD objects created for the Azure Stack HCI deployment. <br> The prefix must not exceed 8 characters.|
 |`-Deploy`|Select this scenario for a brand new deployment instead of an upgrade of an existing system.|-->
 
+> [!NOTE]
+> The `-AsHciOUName` path does not support the following special characters **anywhere** within the OUPath.
+>
```
Comment (alkohli) [inline azure-stack/hci/deploy/deployment-prep-active-directory.md:61]: Change: The `-AsHciOUName` path does not support the following special characters **anywhere** within the OUPath.
**& < > \ '**

To: 
> - The `-AsHciOUName` path doesn't support the following special characters anywhere within the path - `&,”,’,<,>`.
> - Moving the computer objects to a different OU once the deployment is complete, is also not supported.

**Review by alkohli — COMMENTED** [review]
Thank you Steve for your contribution. Please see the suggested changes below and update your PR. The list of unsupported characters is from Thomas.

**Comment by alkohli** [comment]
@steveswalwell Please see the PR for comments. Thank you!

**Comment by alkohli** [comment]
#sign-off

File: azure-stack/hci/deploy/deployment-prep-active-directory.md [hunk 2]
```diff
@@ -56,6 +56,10 @@ The *AsHciADArtifactsPreCreationTool.ps1* module is used to prepare Active Direc
 |`-AsHciDeploymentPrefix`|The prefix used for all AD objects created for the Azure Stack HCI deployment. <br> The prefix must not exceed 8 characters.|
 |`-Deploy`|Select this scenario for a brand new deployment instead of an upgrade of an existing system.|-->
 
+> [!NOTE]
+> - The -AsHciOUName path doesn't support the following special characters anywhere within the path - &,”,’,<,>.
+> - Moving the computer objects to a different OU once the deployment is complete, is also not supported.
```
Comment (Court72) [inline azure-stack/hci/deploy/deployment-prep-active-directory.md:61]:
```suggestion
> - The `-AsHciOUName` path doesn't support the following special characters anywhere within the path `- &,”,’,<,>`.
> - Moving the computer objects to a different OU after the deployment is complete is also not supported.
```

**Pull request merged**